contextual_orchestrator/server.py

```
@@ -1855,10 +1859,40 @@ def _validate_mode(mode: Any) -> str:
+def _validate_capability_request(path: str, body: dict[str, Any]) -> None:
+    """Validate the required trust-boundary fields for media/rerank passthrough."""
+    model = body.get("model")
+    if model is not None and (not isinstance(model, str) or not model.strip()):
+        raise RequestError(400, "invalid_model", "model must be a non-empty string")
+    required_strings = {
+        "/v1/images/generations": ("prompt",),
+        "/v1/videos": ("prompt",),
+        "/v1/audio/speech": ("input", "voice"),
+        "/v1/rerank": ("query",),
+    }.get(path, ())
+    for field in required_strings:
```

Ruff (0.16.2)

[error] Import `field` from line 5 shadowed by loop variable

(F402)